packages/tabsheet/theme/lumo/vaadin-tabsheet-styles.js

```diff
@@ -16,6 +16,10 @@ const tabsheet = css`
     border-radius: var(--lumo-border-radius-l);
   }
 
+  :host([theme~='no-padding'])::part(content) {
```

Comment:
We shouldn't use `::part()` selector inside shadow DOM, use this instead (see line 34 below).

Also, let's place this code block after `[part='content']` to keep content-related rules together.

```suggestion
  :host([theme~='no-padding']) [part='content'] {
```

Reply:
> Also, let's place this code block after [part='content'] to keep content-related rules together.

@FrediWa the selector looks good now 👍  could you also move it closer to the content part, as Serhii suggested?